examples/src/utils/ModelUi.ts

```diff
@@ -174,10 +182,15 @@ export class ModelUi {
   }
 }
 
-async function addLocalModel(viewer: Cognite3DViewer, addModelOptions: AddCdfModelOptions): Promise<CogniteModel> {
+async function addLocalModel(
+  viewer: Cognite3DViewer,
+  addModelOptions: AddModelOptions<DataSourceType>
+): Promise<CogniteModel<DataSourceType>> {
   const isPointCloud =
     addModelOptions.localPath !== undefined && (await isLocalUrlPointCloudModel(addModelOptions.localPath));
-  return isPointCloud ? viewer.addPointCloudModel(addModelOptions) : viewer.addCadModel(addModelOptions);
+  return isPointCloud
+    ? viewer.addPointCloudModel(addModelOptions)
+    : viewer.addCadModel(addModelOptions as ClassicAddModelOptions);
```

Comment:
Do we expect users to type cast `addModelOptions` to `ClassicAddModelOptions`?

Reply:
No... I can change it